src/LibVLCSharp.MAUI/AppHostBuilderExtensions.cs

```diff
@@ -15,7 +17,18 @@ public static MauiAppBuilder UseLibVLCSharp(this MauiAppBuilder builder)
             // Register LibVLCSharp handlers
             builder.ConfigureMauiHandlers(handlers =>
             {
-                handlers.AddHandler(typeof(VideoView), typeof(VideoViewHandler));
+                //todo: add #elif for MacOS...
```

Comment:
no need for this todo :) we will add it when we add macos support. todos always get forgotten (by me anyway)

Reply:
removed TODO

Reply:
macOS has not been started but there is an issue for it on gitlab. Should be straightforwared, PR welcome!